src/utils/satelliteUtils.ts

```diff
@@ -0,0 +1,21 @@
+import { Satellite } from './../db/schema/satellite'
+export default class SatelliteUtilities {
+
+    static async createSatelite(newSatelite: Record<string, string>): Promise<void> {
```

Comment:
`async` and `await` unecessary. 
Just make it a normat function and do `return Satellite.create(newSatelite)`